main/main.go

```diff
@@ -121,6 +121,18 @@ var (
 			Value:  "127.0.0.1:8125",
 			EnvVar: "DATADOG_HOST",
 		}),
+		altsrc.NewStringFlag(cli.StringFlag{
+			Name: config.VAR_NAME_DD_AGENT_HOST,
+			Usage: "datadog statsd agent hostname. This value takes precedence over '" + config.VAR_NAME_DD_HOST +
+				"' if it is provided or found in the env",
+			EnvVar: "DATADOG_AGENT_HOST",
+		}),
+		altsrc.NewStringFlag(cli.StringFlag{
```

Comment:
Should be `IntFlag`

Reply:
I'll change it, but I want to note that the postgres connection port flag is also a string. Should we also keep this new dd agent port flag as a string for consistency?